book/src/sprites/modify_the_texture.md

```diff
@@ -0,0 +1,61 @@
+# Modify The `Texture`
+
+The colours of the sprite will show up exactly as in the source file,
```

Comment:
Should probably go with the American spelling "color" instead of "colour" to be consistent with the api.

Reply:
Will change in next commit, but I think it's worth noting that the Appendix A uses the english spelling in several places - is that worth changing as well?

Reply:
The American spelling should be used everywhere. If you find places that use the British spelling you can change it.

Reply:
Changed everywhere including, slightly concerningly, one .rs file